tools/harness-simulation/harness/install.bat

```diff
@@ -0,0 +1,2 @@
+copy /Y harness\Thread_Harness %systemdrive%\GRL\Thread1.2\Thread_Harness
```

Comment:
Yes, good to have it under thirdparty directory may be `third_party/simulator`

Reply:
Moving to third_party/simulator would require adding a new PYTHONPATH. 
Copying to third_party/ot-comm is more convenient and needs no new PYTHONPATH. 

We can reinterpret ot-comm as ot-common rather than ot-commissioner. 😄